app/database_etl/summary_report_generator/summary_report.py

```diff
@@ -0,0 +1,64 @@
+from time import time
+from app.serotracker_sqlalchemy import db_session, DashboardSource, ResearchSource, \
+    Country, State, City, TestManufacturer, AntibodyTarget, CityBridge, StateBridge, \
+    TestManufacturerBridge, AntibodyTargetBridge
+from sqlalchemy import func, inspect
+from app.utils.notifications_sender import send_slack_message
+import json
+import datetime
+
+class SummaryReport:
+    def __init__(self):
+        self.start_time = time()
+        self.get_table_counts_before()
+
+    def get_elapsed_time(self):
+        return time() - self.start_time
+
+    def get_num_airtable_records(self, num_records: int):
+        self.num_airtable_records = num_records
+
+    def get_table_counts(self):
+        table_counts = {}
+        with db_session() as session:
+            for table in [DashboardSource, ResearchSource, Country, State, City,
+                          TestManufacturer, AntibodyTarget, CityBridge, StateBridge,
+                          TestManufacturerBridge, AntibodyTargetBridge]:
+                # Inspect primary key of table
+                pk = inspect(table).primary_key[0].key
+                # Use primary key to get row count instead of using the .count() method for perf reasons
+                table_counts[table.__tablename__] = session.query(func.count(getattr(table, pk))).scalar()
+        return table_counts
+
+    def get_table_counts_before(self):
+        self.table_counts_before = self.get_table_counts()
+
+    def get_table_counts_after(self):
+        self.table_counts_after = self.get_table_counts()
+
+    def get_human_readable_time(self, timestamp: float):
```

Comment:
i feel like we dont rly need this func cause it's just used once and we can put the logic where it's used

Reply:
down for abstraction, jk ya sure

Reply:
actually I am going to leave this as a function here, it makes the code in send_summary_report look cleaner